src/main/java/com/conveyal/datatools/manager/models/FeedSource.java

```diff
@@ -47,9 +47,17 @@
     /**
      * The collection of which this feed is a part
      */
-    //@JsonView(JsonViews.DataDump.class)
     public String projectId;
 
+    /**
+     * When snapshotting a GTFS feed for editing, gtfs-lib currently defaults to normalize stop sequence values to be
+     * zero-based and incrementing. This can muck with GTFS files that are linked to GTFS-rt feeds by stop_sequence, so
+     * this override flag currently provides a workaround for feeds that need to be edited but do not need to edit
+     * stop_times or individual patterns. WARNING: enabling this flag for a feed and then attempting to edit patterns in
+     * complicated ways (e.g., modifying the order of pattern stops) could have unexpected consequences.
```

Comment:
Seems like it'd be good to have some kind of UI warning for this.

Reply:
oh, just read the PR description. Maybe edit this comment to say that there is no UI setting for this and it is not recommended to do this unless absolutely necessary.